Ex1/week5_1.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import os
+import random
+
+# That's the way
+
+
+def find_files_in_directory_startwith_deep(path):
+    """
+    This function gets the path to a directory and return a list of files that start with the string "deep"
+    :param path: - The path to the directory
+    :return: List of files that start with "deep"
+    """
+    return [file for file in os.listdir(path) if file.startswith("deep")]
+
+
+# I don't have vinaigrette
+
+
+def get_random_date_between(date1, date2):
+    """
+    The function gets 2 dates and return a random date between those dates
+    :param date1: a date from the input in format "YYYY-MM-DD"
+    :param date2: a date from the input in format "YYYY-MM-DD"
+    :return: random date between the dates
+    """
+    random_number_of_days = random.randrange(abs(date1 - date2).days)
+    random_date = date1 + datetime.timedelta(random_number_of_days)
+    return str(random_date)
+
+
+def random_date_between_dates_check_if_monday(date1, date2):
```

Comment:
Use type hints instead of documenting the variable types - https://stackoverflow.com/questions/5336320/how-to-know-function-return-type-and-argument-types